data/metric/metrics.go

```diff
@@ -25,48 +32,75 @@ const (
 	ATTRIBUTE SourceType = iota
 )
 
+const (
+	// NSSeparator is the metric namespace separator
+	NSSeparator = "::"
+	// NSAttributeSeparator is the metric attribute key-value separator applied to generate the metric ns.
+	NSAttributeSeparator = "=="
```

Comment:
Shall we use `|` just to keep consistency with our entity GUID spec?

Reply:
This is just a disk-storage implementation detail. Nothing related to be managed by user, neither to be pushed to agent. So entity naming here would be indeed misleading.

Reply:
👍

Reply:
What I'd do is privatize these consts